### PR TITLE
refactor strings to bytes

### LINKED ISSUE #171: Fix proto def's for pub key types
- update all pub key / address types in proto definitions to use byte[] vs string (for example - https://github.com/arkeonetwork/arkeo/blob/817a8e1e34e29e1ab3098f065c3a6107dcb0abd6/proto/arkeo/arkeo/keeper.proto#L16)
- ensure all tests past with changes

### PR DESCRIPTION
**Pull request by cbarraford**
Fixes #171

### CODE REVIEW

File: arkeocli/claim.go [hunk 1]
```diff
@@ -69,15 +69,15 @@ func runClaimCmd(cmd *cobra.Command, args []string) (err error) {
...
+	if clientPubkey.IsEmpty() {
 		clientPubkey = contract.GetClient()
 	}
 
 	creatorAddr, err := clientPubkey.GetMyAddress()
 	if err != nil {
 		return err
 	}
-	creator := creatorAddr.String()
+	creator := creatorAddr
```
Comment (0xean) [inline arkeocli/claim.go:80]: not sure I am seeing why this assignment is needed now?

File: proto/arkeo/claim/tx.proto [hunk 2]
```diff
@@ -13,31 +14,31 @@ service Msg {
...
 }
 
 message MsgTransferClaimResponse {}
 
 message MsgAddClaim {
-  string creator = 1;
+  bytes creator = 1 [ (gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress" ];
   Chain chain = 2;
-  string address = 3;
+  bytes address = 3 [ (gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress" ];
```
Comment (0xean) [inline proto/arkeo/claim/tx.proto:38]: not a huge deal, but for this one, the address here can also be an ETH addy.  this is just for testing purposes as this TX type only works on testnet, but may be nice functionality when people are developing the claim UI so we don't have to worry about creating claims on testnet and people can self serve.

**Review by 0xean — CHANGES_REQUESTED** [review]
minor comments left, amazing how much boilerplate this got rid of, I like it.

File: arkeocli/claim.go (hunk 1 above)
Comment (cbarraford) [reply arkeocli/claim.go]: yup, removed.

**Review by 0xean — APPROVED** [review]
LGTM, thanks

**Pull request merged**